lib/bolt/shell/powershell/snippets.rb

```diff
@@ -55,21 +55,25 @@ def run_script(arguments, script_path)
             }
             #{build_arg_list}
 
-            switch -regex ( Get-ExecutionPolicy )
+            try
```

Comment:
@mcdonaldseanp I think the path is messed up for module loading in our test runners:

```
       +"merged_output" => "Get-ExecutionPolicy : The 'Get-ExecutionPolicy' command was found in the module 'Microsoft.PowerShell.Security', but \r\nthe module could not be loaded. For more information, run 'Import-Module Microsoft.PowerShell.Security'.\r\nAt line:10 char:17\r\n+ switch -regex ( Get-ExecutionPolicy )\r\n+                 ~~~~~~~~~~~~~~~~~~~\r\n    + CategoryInfo          : ObjectNotFound: (Get-ExecutionPolicy:String) [], CommandNotFoundException\r\n    + FullyQualifiedErrorId : CouldNotAutoloadMatchingModule\r\n \r\nparam\n",
```
I think that the intent of this was simply to provide a helpful error message and that we should not error if we cant load that module. In practice i doubt this comes up much because task running all works due to other powershell setup we do. 

Do you think a `catch` here is reasonable, should it be more specific on objects it catches?

Reply:
https://github.com/puppetlabs/bolt/issues/2659 for reference